code/obj/item/clothing/injector_mask_belt.dm

```diff
@@ -414,7 +414,7 @@ There's A LOT of duplicate code here, which isn't ideal to say the least. Should
 		return 1
 
 	check_trigger(mob/M)
-		if(M.getStatusDuration("stunned") || M.getStatusDuration("paralysis") || M.getStatusDuration("weakened") || isunconscious(M)) return 1
+		if(!can_act(M, 0)) return 1
```

Comment:
I want to point out that as it is the new code will trigger on death when the old one didn't, but I don't expect it to be an important difference. Dying without going unconscious first is somewhat of a tall order anyway.